pages/common/duplicity.md

```diff
@@ -0,0 +1,23 @@
+# duplicity
+
+> Creates incremental, compressed, encrypted, and versioned backups and optionally uploads them to a variety of backend services.
+
+- Backup a folder via ftps to a remote machine, encrypting with a password:
+
+`FTP_PASSWORD={{ftp_login_password}} PASSPHRASE={{encryption_password}} duplicity {{path/to/source/directory}} {{ftps://user@remotehost.tld/target/directory/path/}}`
```

Comment:
Sorry, they're required! and `FTP_PASSWORD` is actually a duplicity thing to pass a password to a backend. While I do agree they clutter the example, unfortunately they are required 😕

Sure, I'll change that to `hostname`.